src/views/GovernanceReviewTeam/BusinessCaseReview/index.tsx

```diff
@@ -14,6 +15,7 @@ type BusinessCaseReviewProps = {
 };
 const BusinessCaseReview = ({ businessCase }: BusinessCaseReviewProps) => {
   const { t } = useTranslation('governanceReviewTeam');
+  const filename = `Business case for ${businessCase.requestName}.pdf`;
```

Comment:
I question whether this oughta be in i18n? It's english text, but it's a filename, so idk.

Reply:
That's a good callout. Let's save that for some cleanup (along with the padding).